cli/src/cli.rs

```diff
@@ -395,38 +378,54 @@ pub async fn change_password_command(wallet: &Wallet, current_password: Password
     Ok(())
 }
 
-pub async fn init_command(
-    storage_path: &Path,
-    secret_manager: SecretManager,
-    init_params: InitParameters,
-) -> Result<Wallet, Error> {
-    let alias = if get_decision("Do you want to assign an alias to your wallet?")? {
-        Some(get_alias("New wallet alias").await?)
-    } else {
-        None
-    };
+pub async fn init_command(storage_path: &Path, init_params: InitParameters) -> Result<Wallet, Error> {
+    let secret_manager = create_secret_manager(&init_params).await?;
+    let secret_manager_variant = secret_manager.to_string();
+
+    let mut address = init_params.address.map(|s| Bech32Address::from_str(&s)).transpose()?;
+    let mut forced = false;
+    if address.is_none() {
+        if enter_decision("Do you want to set the address of the new wallet?")? {
```

Comment:
`enter_decision()` has default `yes` which should be wrong for the address I'd say

Reply:
Changed